lib/config/locales/en.yml

```diff
@@ -373,9 +373,9 @@ en:
           resolution: "Set the values from the parent, or redefine the association
             with only a single definition in the parent."
         invalid_value:
-          message: "Value of type %{value_class} cannot be written to a field of type %{field_class}"
-          summary: "Tried to set a value of type %{value_class} to a field of type %{field_class}"
-          resolution: "Verify if the value to be set correspond to field definition"
+          message: "The value %{value} cannot be written to a field of type %{field_class}."
+          summary: "The value %{value} could not be coerced to type %{field_class}."
+          resolution: "Verify that the value to be set corresponds to the field definition."
```

Comment:
This error is most likely no longer needed, and can be repurposed for raising when `strict: true`

Reply:
The error should be removed from this PR then. I'd like to ask that we think further around `strict: true` and consider other alternatives; we can add the error back when it is actually needed.

Reply:
done